.docker/build.Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Define ARG we use through the build
+ARG VERSION=unstable
```

Comment:
Either you must pass stable as argument in the workflows or change it here to stable
```suggestion
ARG VERSION=stable
```

Reply:
Still unstable and should be stable.